explorer/src/Pos/Explorer/Socket/App.hs

```diff
@@ -3,6 +3,7 @@
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE TemplateHaskell     #-}
 {-# LANGUAGE TypeOperators       #-}
+{-# LANGUAGE GADTs               #-}
```

Comment:
🤔

Reply:
GADTs is needed for the `~` syntax

Reply:
It's true, but there is no `~` syntax in this file.

Reply:
Hm-m-m.